src/assets/i18n/fi.json

```diff
@@ -0,0 +1,51 @@
+	"About domain name system, DNS": "Nimipalvelujärjestelmä (DNS)",
+	"Algorithm": "Algoritmi",
+	"Choose at least one protocol": "Valitse vähintään toinen vaihtoehdoista IPv4 ja IPv6.",
+	"Digest": "Yhteenveto",
+	"Digest required": "Yhteenveto tarvitaan",
+	"Digest type": "Yhteenvedon tyyppi",
+	"DS_RECORDS": "Delegation Signers (DS-poster)",
+	"Domain check": "Verkkotunnustesti",
+	"Domain checked completed": "Verkkotunnustesti valmis",
+	"Domain name": "Verkkotunnus",
+	"Domain name required": "Verkkotunnus ilmoitettava",
+	"Error during parent data fetching": "Virhe haettaessa dataa pääverkkotunnuksesta",
+	"Export": "Vie",
+	"FAQ": "Usein kysytyt kysymykset",
+	"Fetch data from parent zone": "Hae dataa pääverkkotunnuksesta",
+	"Filter text": "Suodata viestit",
+	"Group by modules": "Jaa ryhmiin moduulien perusteella",
+	"History": "Historia",
+	"History information request is in progress": "Aiempien testitulosten haku käynnissä",
+	"Internal server error": "Sisäinen järjestelmävirhe",
+	"Key Tag": "Key Tag",
+	"Keytag required": "Keytag tarvitaan",
+	"Level": "Taso",
+	"Link": "Linkki",
+	"Message": "Viesti",
+	"Module": "Moduuli",
+	"Nameservers": "Nimipalvelin",
+	"No data for this test": "Ei dataa tätä testiä varten",
+	"No result for this query": "Testi ei tuottanut tulosta",
+	"Notice! More info on undelegated test": "Ilmoitus! Lisätietoja delegoimattomasta testistä",
+	"Options": "Options",
+	"Page Not Found": "Sivua ei löytynyt",
+	"Parent data fetched with success": "Data haettiin pääverkkotunnuksesta ongelmitta",
+	"Please set the api endpoint": "Aseta api endpoint",
+	"Program versions": "Ohjelman versiot",
+	"Reset": "Tyhjennä valinnat",
+	"Results from previous tests are not available": "Ei aiempia testituloksia",
+	"Run test": "Suorita testi",
+	"Settings": "Asetukset",
+	"Share": "Jaa",
+	"Sorry, there is nothing here!": "Ikävä kyllä täällä ei ole mitään!",
+	"Test #": "Testinumero ",
+	"There is no delegation for the zone": "Verkkotunnuksessa ei ole delegointia",
+	"What is an undelegated domain test?": "Mikä on delegoimaton verkkotunnustesti? ",
+	"Zonemaster Backend is not available": "Zonemaster Backend ei ole käytettävissä"
+  ,"Profile"                                : "Profiili"
```

Comment:
The alignment looks odd. But looking at the other translated files, they all have the same appearance for this 2 lines.

Reply:
This is identical with previous parts:

"Zonemaster Backend is not available": "Zonemaster Backend ei ole käytettävissä",
"Profile" : "Profiili",

Reply:
@pnax, all the XX.json look that way. I do not know why, but it does not seem to matter.

Reply:
Yes I noticed this as well. I opened #186 to fix this.

Reply:
Fixed in new commit.